.circleci/config.yml

```diff
@@ -81,7 +84,7 @@ jobs:
 
   Build-Docs:
     docker:
-      - image: circleci/python:3.7
```

Comment:
not sure, maybe this is causing the docs build error. 
btw why this change?

Reply:
because `circleci/python:3.7` does not have `latex` and latex is needed for imgmath extension

Reply:
so the problem is in python version, py3.6 does not support the inheritance... the min version is 3.7